src/app/_components/CardLayout.tsx

```diff
@@ -17,6 +17,7 @@ export function CardLayout({
     blogPost: 'lg:grid-cols-2 lg:grid-rows-2',
     caseStudy: 'md:grid-cols-2 md:grid-rows-3 lg:grid-cols-3 lg:grid-rows-2',
     reports: 'lg:grid-cols-2 lg:grid-rows-2',
+    grants: 'sm:grid-cols-3 sm:gap-6 gap-7',
```

Comment:
You can use `Tailwind: Sort Selection` to sort these. Need to select them first.

<img width="712" alt="Screenshot 2024-04-29 at 10 21 16" src="https://github.com/FilecoinFoundationWeb/filecoin-foundation/assets/26620750/beb59bed-e35f-4486-a5a4-b72cdb7fb362">